src/LiveComponent/src/Attribute/AsLiveComponent.php

```diff
@@ -28,7 +28,8 @@ public function __construct(
         bool $exposePublicProps = true,
         string $attributesVar = 'attributes',
         public bool $csrf = true,
-        public string $route = 'ux_live_component'
+        public string $route = 'ux_live_component',
+        public bool $forcePost = false
```

Comment:
As it's almost a classic controller, why not use `method: ['GET', 'POST']` ?   as in #[AsController] ?

Reply:
The `RequestBuilder` in the JS part has a more advanced logic to switch between GET and POST methods. Therefore I think using `method` would be misleading.